docker-compose.yml

```diff
@@ -23,39 +23,38 @@ services:
     environment:
       ORB_PORT: '10000'
       TANGO_HOST: databaseds.workspace:10000
-      MYSQL_HOST: mysql.workspace:3306
+      MYSQL_HOST: mariadb.workspace:3306
       MYSQL_DATABASE: tango_db
       MYSQL_USER: tango
       MYSQL_PASSWORD: tango
     volumes:
       - ./.config/supervisord:/etc/supervisor/init.d:ro
     links:
-      - mysql:mysql.workspace
+      - mariadb:mariadb.workspace
 
   tangotest:
-    image: tangocs/tango-cs:9
+    image: synchrotronsolaris/centos-tango-dev:latest
     hostname: tangotest.workspace
     command: >-
       /bin/sh -c 'wait-for-it.sh $${TANGO_HOST} --timeout=30 --strict --
       supervisord -c /etc/supervisor/init.d/supervisord.tangotest.conf'
     environment:
+      ORB_PORT: 10000
       TANGO_HOST: databaseds.workspace:10000
     volumes:
       - ./.config/supervisord:/etc/supervisor/init.d:ro
     links:
       - databaseds:databaseds.workspace
 
-  mtango:
-    image: mliszcz/mtango:rc3-0.1
-    hostname: mtango.workspace
-    command: >-
-      /bin/sh -c 'wait-for-it.sh $${TANGO_HOST} --timeout=30 --strict --
-      supervisord -c /etc/supervisor/init.d/supervisord.mtango.conf'
+  taurus:
+    image: synchrotronsolaris/centos-taurus:latest
+    hostname: taurus.workspace
     environment:
       TANGO_HOST: databaseds.workspace:10000
       USER_NAME: tango
       USER_PASSWORD: tango
       HTTP_PORT: '9091'
```

Comment:
Is this `HTTP_PORT` needed here? I think it's a leftover from mtango configuration.